docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -303,29 +303,27 @@ You can only edit the settings of Data Assets created in GX Cloud.
+To connect to your database in GX Cloud, there are two methods for managing credentials:
 
-1. Store your credential value as an environment variable by entering `export ENV_VAR_NAME=env_var_value` in the terminal or adding the command to your `~/.bashrc` or `~/.zshrc` file. For example:
+1. **Direct input**: You can input credentials directly into GX Cloud. These credentials are securely encrypted at rest and in transit, stored in our database, and accessible to the GX Agent and any API user.
 
-    ```bash title="Terminal input"
-    export GX_CLOUD_SNOWFLAKE_PASSWORD=<password-string>
-    ```
-    Prefix environment variable names with `GX_CLOUD_`.
+2. **Environment variable substitution**: To enhance security, you can use environment variables to manage sensitive connection parameters or strings. For example, instead of directly including your database password in configuration settings, you can use a variable reference like `${MY_DATABASE_PASSWORD}`:
 
-2. Create a Data Source connection string using the environment variable name instead of the credential value. For example:
+    - **Configure the environment variable**: Enter the name of your environment variable, enclosed in `${}`, into the UI configuration form. For instance, you might use `${MY_DATABASE_PASSWORD}`.
 
-    ```python title="Example Data Source connection string"
-    snowflake://<user-name>:${GX_CLOUD_SNOWFLAKE_PASSWORD}@<account-name>/<database-name>/<schema-name>?warehouse=<warehouse-name>&role=<role-name>
+    - **Inject the variable into your agent container**: When setting up your Docker container, include the environment variable in the command. For example:
+   
+   ```bash title="Terminal input"
+   docker run -it -e MY_DATABASE_PASSWORD=<YOUR_DATABASE_PASSWORD> -e GX_CLOUD_ACCESS_TOKEN=<YOUR_ACCESS_TOKEN> -e GX_CLOUD_ORGANIZATION_ID=<YOUR_ORGANIZATION_ID> greatexpectations/agent:stable
     ```
-    Environment variable names must be enclosed by curly braces and be preceded by a dollar sign. For example: `${GX_CLOUD_SNOWFLAKE_PASSWORD}`. Do not use interpolation to add credential values to connection strings.
 
-3. Use the environment variable to supply the credential value when you run the GX Agent. For example:
+This process is similar across any Docker-based service, including Kubernetes, ECS, ACI, and GCE. Each platform will have its own method for setting environment variables.
 
-    ```bash title="Terminal input"
-    docker run --rm -e GX_CLOUD_SNOWFLAKE_PASSWORD="<snowflake_password>" -e GX_CLOUD_ACCESS_TOKEN="<user_access_token>" -e GX_CLOUD_ORGANIZATION_ID="<organization_id>" greatexpectations/agent
-    ```
+With this method, your password is not stored or transmitted to GX Cloud. To access the database through the API, just set the environment variable in your setup. 
+
+**Environment variable substitution is not supported in fully hosted deployments.**
```

Comment:
I'd recommend moving this sentence to directly below the `2. **Environment variable substitution**:` paragraph (before the `Configure the environment bullet`) and making it a warning admonition box with a title of "Environment variable substitution support").

https://docusaurus.io/docs/markdown-features/admonitions